src/packages/blits/.storybook/preview.ts

```diff
@@ -0,0 +1,45 @@
+import Blits from '@lightningjs/blits';
+import type { Preview } from '@storybook/html';
+
+
+const launchApp = ({name, fn, template}) => {
+
+// app with single component
+const App = Blits.Application({
+  components: {
+    [name]: fn
+  },
+  template,
+})
+
+// timeout to wait until app div is available
+setTimeout(() => {
```

Comment:
Perhaps we might be able to use MutationObserver here to wait for the div to be added to the DOM?

Reply:
Thanks Josh good idea I've just pushed a change for that [here](https://github.com/lightning-js/ui-components/pull/128/commits/ea4017d352916be1749030dc1079e6ed246f0e6a)